Form1.cs

```diff
@@ -6093,7 +6093,19 @@ private async void actionTimer_TickAsync ( object sender , EventArgs e )
                               }
                               else if ( ( Form2.config.plBlink ) && ( !plStatusCheck ( StatusEffect.Blink ) ) && ( CheckSpellRecast ( "Blink" ) == 0 ) && ( HasSpell ( "Blink" ) ) )
                               {
-                                   CastSpell ( "<me>" , "Blink" );
+                                    if (Form2.config.Accession && Form2.config.blinkAccession && currentSCHCharges > 0 && HasAbility("Accession") && !plStatusCheck(StatusEffect.Accession))
+                                    {
+                                        JobAbility_Wait("Blink, Accession", "Accession");
+                                        return;
+                                    }
+
+                                    if (Form2.config.Perpetuance && Form2.config.blinkPerpetuance && currentSCHCharges > 0 && HasAbility("Perpetuance") && !plStatusCheck(StatusEffect.Perpetuance))
+                                    {
+                                        JobAbility_Wait("Blink, Perpetuance", "Perpetuance");
+                                        return;
+                                    }
+

```

Comment:
These lines I just copied from another entry. For whatever reason, even though blink had perp and acces buttons, it had no code to check those buttons.